src/python/CMSSpark/schemas.py

```diff
@@ -561,3 +561,64 @@ def schema_empty_eos():
         StructField("timestamp", LongType(), True),
         StructField("day", LongType(), True),
     ])
+    
+def schema_cmspop_json():
+    return StructType(
+        [
+            StructField(
+                "data",
+                StructType(
+                    [
+                        StructField("app_info", StringType(), True),
+                        StructField("client_domain", StringType(), True),
+                        StructField("client_host", StringType(), True),
+                        StructField("end_date", StringType(), True),
+                        StructField("end_time", LongType(), True),
+                        StructField("fallback", BooleanType(), True),
```

Comment:
WE just discussed in [this issue](https://cern.service-now.com/service-portal/view-request.do?n=RQF1566581) that fallback attribute may be boolean or string, therefore it would be better if you adjust it to StringType here.